domains/tommyc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "FlipMods",
+        "email": "tommy.camp@ccsstudent.org"
+    },
+    "record": {
+        "CNAME": ["https://pf-rouge.vercel.app/"]
```

Comment:
1. Vercel is not supported
2. `CNAME` can't be in an array
3. `CNAME` should not contain protocol (`https://`)